### ISSUE TITLE
Map Vault API

### ISSUE THREAD

**Issue by IDragonfire** [FAForever/api#6]
* Last Maps/Updates
* Newest upcoming
* Map RESTFul API
 * index
 * show
 * rate
 * report
 * manage/update

@Sheeo 
Should user comment a map?
I suggest to move that to the forum ;)

**Comment by Sheeo** [comment]
I don't mind allowing user comments on maps, we may need to ensure we get a feed of comments for moderation, though.

**Comment by IDragonfire** [comment]
Yes, but why write our own system?
My suggestion: 
Use our new forum for that ...
We need that also for mods ;)

**Comment by aeoncleanse** [comment]
Is this all done now?

**Comment by micheljung** [comment]
No, what makes you think it's done? Issue is open, no milestone, no assignee, no commits, no pull request, no code in the repository... sometimes I just don't understand :P
